src/vs/workbench/contrib/debug/browser/repl.ts

```diff
@@ -811,3 +812,27 @@ export class ClearReplAction extends Action {
 function getReplView(viewsService: IViewsService): Repl | undefined {
 	return viewsService.getActiveViewWithId(REPL_VIEW_ID) as Repl ?? undefined;
 }
+
```

Comment:
This makes sense to live in this file, since it can not appear anywhere else.
I also suggest to completyl remove `annotationForeground` because due to a bug we do not show it. Until we fix that I suggest to not make this themable.

Reply:
Okay, I will remove that contribution. Should I restore the original CSS for the annotation so it is not forgotten when "the time is right"?

Reply:
Yeah you can restore the original css